src/analyzer.ts

```diff
@@ -269,8 +264,8 @@ export class Analyzer {
           }
         });
 
-    const dependencies =
-        (await Promise.all(scannedSubDocuments)).filter(s => !!s);
+    const dependencies = (await Promise.all(scannedSubDocuments))
+                             .filter(s => !!s) as ScannedDocument[];
```

Comment:
Not without sacrificing local correctness, because we're only filtering out the nulls here.